### DISCUSSION TITLE
Score the snooker game

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1074] [Galaxy Trivia] [answered]
🔴🔵🔴⚫🔴⚫🔴⚫🔴⚫🔴🔵🔴🔵🔴🕳️🔴⭕🔴⭕🔴🟢🔴⚫🔴⭕🔴⚫🟡🟢🟤🔵⭕⚫

**Bonus:** Is there a century break in the game?

**Comment by thuongtruong1009** [comment, marked as answer]
To calculate the score of the snooker game, we need to assign point values to each colored ball. In snooker, the point values are as follows:

- Red ball: 1 point
- Yellow ball: 2 points
- Green ball: 3 points
- Brown ball: 4 points
- Blue ball: 5 points
- Pink ball: 6 points
- Black ball: 7 points

Now let's count the scores for each color:

🔴 (Red) - 1 point
🔵 (Blue) - 5 points
🔴 (Red) - 1 point
⚫ (Black) - 7 points
🔴 (Red) - 1 point
⚫ (Black) - 7 points
🔴 (Red) - 1 point
⚫ (Black) - 7 points
🔴 (Red) - 1 point
⚫ (Black) - 7 points
🔴 (Red) - 1 point
🔵 (Blue) - 5 points
🔴 (Red) - 1 point
🔵 (Blue) - 5 points
🔴 (Red) - 1 point
🕳️ (Miss) - 0 points (No score deduction for a miss)
🔴 (Red) - 1 point
⭕ (White) - 0 points (White ball - no score)
🔴 (Red) - 1 point
⭕ (White) - 0 points (White ball - no score)
🔴 (Red) - 1 point
🟢 (Green) - 3 points
🔴 (Red) - 1 point
⚫ (Black) - 7 points
🔴 (Red) - 1 point
⭕ (White) - 0 points (White ball - no score)
🔴 (Red) - 1 point
⚫ (Black) - 7 points
🟡 (Yellow) - 2 points
🟢 (Green) - 3 points
🟤 (Brown) - 4 points
🔵 (Blue) - 5 points
⭕ (White) - 0 points (White ball - no score)
⚫ (Black) - 7 points

Now let's calculate the total score:

Total Score = 1 + 5 + 1 + 7 + 1 + 7 + 1 + 7 + 1 + 7 + 1 + 5 + 1 + 5 + 1 + 0 + 1 + 0 + 1 + 0 + 1 + 3 + 1 + 7 + 1 + 0 + 1 + 7 + 2 + 3 + 4 + 5 + 0 + 7
Total Score = 76 points

Now, to determine if there's a century break in the game, we need to see if any player has scored 100 points or more in a single turn. In this case, the total score is 76 points, which means there is no century break in the game. A century break would require the player to score at least 100 points in one turn without the opponent having a chance to play.

**Reply by ParanoidUser** [reply]
Great job, @thuongtruong1009! How would the score change if ⭕ was a pink ball? (*this was the closest I got to pink*)

**Comment by peterbaccaro** [comment]
The total points scored in the game was **119**.

The first player would have scored **51** points before they missed and then the second player would have scored **68** points.

Therefore the game score would be as follows:
- Player 1: **51**
- Player 2: **68**

Given that neither player scored at least 100 points in a single turn there was **no century break in the game**.
